### PR TITLE
Added documentation of CSV input files

### LINKED ISSUE #9472: Add documentation of CSV-input files for models with top-level inputs
This feature is currently undocumented, except for the [-csvInput](https://www.openmodelica.org/doc/OpenModelicaUsersGuide/latest/simulationflags.html#simflag-csvinput) Simulation Flag, whose documentation doesn't tell much. 

It should be described in the User's Guide. I guess we could add it [here](https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/omedit.html#simulating-a-model), though this actually also applies for simulations that are not started from OMEdit.

Keeping @MohammadHadiAlizadeh in the loop.

### PR DESCRIPTION
**Pull request by casella**
Closes #9472.

### CODE REVIEW

**Review by adeas31 — APPROVED** [review]

File: doc/UsersGuide/source/omedit.rst [hunk 1]
```diff
@@ -674,6 +674,25 @@ Here are some simple examples:
...
+~~~~~~~~~~~~~~~~~~~
+When simulating Modelica models with top-level inputs (input variables or input connectors), these inputs are assumed to be zero by default. However,
+it is possible to feed them with input signals obtained from CSV (Comma-Separated Value) input data files, by means of the
+:ref:`-csvInput <simflag-csvInput>` simulation flag, that can be set in the *Additional Simulation Flags (Optional)* field of
+the Simulation Flags tab. For example, setting ``-csvInput=myinput.csv`` causes the runtime executable to read such input data from the ``myinput.csv``
+file.
+
+CSV files should contain the names of the input variables in the first row, beginning with ``time`` on the first column, and the values of such variables
+for each point in time in subsequent rows, with non-decreasing time values. The variable names should be enclosed by quotation marks in case they contain spaces, to avoid ambiguities. The most commonly used separator for data within each row is the comma, but it is also possible to use other separators, e.g., space, tab,
+or semi-colon; the used type of separator will be determined automatically.
```
[the remark this answers is not in the document]
Comment (casella) [reply doc/UsersGuide/source/omedit.rst:687]: Ach, so. I understood from the previous source code that we were just doing that. Isn't it just a matter of figuring out what character comes after `time` or `"time"` at the beginning of the file?

[the remark this answers is not in the document]
Comment (adeas31) [reply doc/UsersGuide/source/omedit.rst:687]: Specifying `sep=x` is more of a csv file standard format.

[the remark this answers is not in the document]
Comment (mahge) [reply doc/UsersGuide/source/omedit.rst:687]: We can do that if you guarantee that it always starts with `time` or `"time"`. It should not be difficult to adjust it. But I think it is a slippery slope.

I did not change the logic btw; it just fixed a bug which prevented it working as expected. It has always been like that (>8 years.)

[the remark this answers is not in the document]
Comment (casella) [reply doc/UsersGuide/source/omedit.rst:687]: Let's not change that, I understand the "sep=x" is more or less a standard. I'll update the documentation accordingly.

**Comment by casella** [comment]
I fixed the documentation with the "sep=x" thing and added some examples. I hope I got the RST right, maybe you can double-check that 😅

**Comment by casella** [comment]
@adeas31 I'm going ahead with the commit, so I can check the final appearance. If you see any issue I can open another PR for that.

**Pull request merged**